src/eq_schema/Block.js

```diff
@@ -8,16 +9,63 @@ const pageTypeMappings = {
 };
 
 class Block {
-  constructor(title, description, page) {
+  constructor(title, description, page, ctx) {
     this.id = `block${page.id}`;
     this.title = getInnerHTML(title);
     this.description = getInnerHTML(description);
     this.type = this.convertPageType(page.pageType);
-    this.questions = this.buildQuestions(page);
+    this.questions = this.buildQuestions(page, ctx);
+    if (!isNil(page.routingRuleSet)) {
+      // eslint-disable-next-line camelcase
+      this.routing_rules = this.buildRoutingRules(
+        page.routingRuleSet.routingRules,
+        page.routingRuleSet,
+        page.id,
+        ctx
+      );
+    }
   }
 
-  buildQuestions(page) {
-    return [new Question(page)];
+  buildQuestions(page, ctx) {
```

Comment:
`Question` passes `ctx` down to `Answer` but `Answer` doesn't use `ctx` at all. So do we need to pass `ctx` down here?